web/src/user_group_create.ts

```diff
@@ -184,6 +189,7 @@ function create_user_group(): void {
         can_join_group: JSON.stringify(can_join_group),
         can_mention_group,
```

Comment:
I guess `can_mention_group` is a variant here because it's not been converted to use the `GroupSettingValue` system yet.